### ISSUE TITLE
Winetricks ie8 download not working

### ISSUE THREAD

**Issue by Ads20000** [Winetricks/winetricks#671]
Winetricks won't download ie8 correctly.
I get the following error if I try to install ie8 with Winetricks (with the GUI) 'sha1sum mismatch! Rename /home/adam/.cache/winetricks/msls31/InstMsiW.exe and try again.'
I have renamed the file and tried again but get the same error.

**Comment by slyscorpion** [comment]
Can you try the following command: 

winetricks --self-update

and then attempt to install ie8 after updating winetricks?

**Comment by Ads20000** [comment]
'unknown option --self-update'
The Ubuntu version of Winetricks is 20140817 so should I manually update Winetricks using [these instructions](http://askubuntu.com/questions/755059/how-do-i-get-the-latest-version-of-winetricks-on-ubuntu)? I don't get a launcher item for Winetricks if I use them though and I suspect it's not installed properly.

**Comment by slyscorpion** [comment]
Yes, definitely follow those instructions as your current version of winetricks is rather old and that is probably part of the reason why you are experiencing issues with ie8. 

So.... update your winetricks by following the instructions you linked and try installing ie8 again. Make sure to set your version of Windows in Wine to be Windows XP and you should be able to install ie8.

**Comment by Ads20000** [comment]
Yeah, I get an 'Internet Explorer 8 is not supported on this operating system' error when I try to install it, then it says it's starting it but fails.

**Comment by slyscorpion** [comment]
@Ads20000  

Could you do the following: 

Run "winetricks --version" and post your version here AFTER updating to the latest version of winetricks and post the version number in this bug report.

Run "winetricks ie8" and please post the terminal output you get so that the maintainer of winetricks can see if it's an issue with winetricks or your Wine set up.

Also, it would help to mention your Wine prefix setup (if you are using any other winetricks, what Windows version you have it set to, stuff like that).

**Comment by Ads20000** [comment]
My Winetricks version is now 20160724.
[Terminal output for command `winetricks ie8`](http://paste.ubuntu.com/23234876/)
After this line 'fixme:ntdll:EtwRegisterTraceGuidsW   register trace class {e2821408-c59d-418f-ad3f-aa4e792aeb79}' I got a window pop up saying 'Internet Explorer is not supported on this operating system'. Dismissing that seemed to allow it to install but running the command suggested causes this to happen:

```
$ wine 'C:\Program Files (x86)\Internet Explorer\iexplore'
wine: cannot find 'C:\Program Files (x86)\Internet Explorer\iexplore'
```

If I manually go to the location I can only see an iexplore.exe.bak and double-clicking on iexplore.exe doesn't have an effect. If I run the command suggested before again I get a slightly different error:

```
$ wine 'C:\Program Files (x86)\Internet Explorer\iexplore'
wine: cannot find L"C:\\Program Files (x86)\\Internet Explorer\\iexplore.exe"
```

Navigating to the directory in .wine and running the command `wine iexplore.exe` doesn't work, with the same error as previous.

No other Winetricks workarounds have been used. Wine's Windows version is Windows XP.

**Comment by slyscorpion** [comment]
@Ads20000  

One more tip, if you don't mind :) 

Always post the winetricks version like so: 

20160724 - sha1sum: 263130014d4ca7ea29c0b345a86115bda93f58a1

because the date "20160724" doesn't change often but the sha1sum does every time winetricks is updated :)

**Comment by Ads20000** [comment]
```
$ winetricks --version
20160724 - sha1sum: 263130014d4ca7ea29c0b345a86115bda93f58a1
```

Thanks for helping to triage this bug/problem I'm having :)

**Comment by slyscorpion** [comment]
@Ads20000 

Like your terminal output states, please try "winetricks ie8" in a 32-bit prefix :)

**Comment by Ads20000** [comment]
OK, that worked (well, not to say that Wine ie8 itself is buggy and LoL still doesn't seem to install with the dependencies suggested on AppDB - I could use POL but then I can't submit to AppDB and I know Wine wants to be able to work without third-party wrappers!), but despite the Terminal warning me, isn't the successful installation of ie8 on 64-bit Wine still a missing feature?

**Comment by slyscorpion** [comment]
Personally, I would close the issue due to the following: 

1. it's up to the maintainer to decide if ie8 on WinXP 64 is a missing feature.

2. Windows XP 64-bit was Microsoft's bastard child OS so it never got the proper support like its later 64-bit systems did. Perhaps there is not a 64-bit version of ie8, I don't know... 

3. The program informs the user that the given winetrick may not work in a 64-bit prefix.

4. Even if you did get ie8 working in a 64-bit prefix then what about LoL? Does *that* application have a 64-bit version? If it's 32-bits only then it will not work in a pure 64-bit prefix. 

So, in short, I would close the ticket with "Invalid" or however you can close an issue on github, then if I were in your place I would try installing the recommended winetricks BEFORE installing what I assume is League of Legends, seeing if it breaks, and then file a bug report in WineHQ if it doesn't work even after you have installed the prerequisite winetricks or whatnot.
